### DISCUSSION TITLE
junixsocket/dependency

### DISCUSSION THREAD

**Discussion by giscus** [kohlschutter/junixsocket#150] [Giscus]
# junixsocket/dependency

https://kohlschutter.github.io/junixsocket/dependency.html

**Comment by moaxcp** [comment]
Should add some info for using modules. I am getting an error with `requires transitive org.newsclub.net.unix;`:

```
module not found: com.kohlschutter.annotations.compiletime
```

**Reply by kohlschuetter** [reply]
Yes, this is unfortunately a known bug that will be fixed in 2.9.0.
For the meantime, you could add 

```
        <dependency>
            <groupId>com.kohlschutter</groupId>
            <artifactId>compiler-annotations</artifactId>
            <version>1.6.5</version>
            <scope>provided</scope>
        </dependency>
```
